### PR TITLE
More Noes & Yessses without swearing

### PR DESCRIPTION
**Pull request by guyhughes**

### CODE REVIEW

File: yes.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
 	"Certainly",
 	"Cool",
 	"Correct",
+	"Damn straight",
+	"Definitely",
 	"Definitly",
```
Comment (sindresorhus) [inline yes.json]: oops, looks like i wrote this incorrectly. can you fix this one instead.

File: yes.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
 	"Certainly",
 	"Cool",
 	"Correct",
+	"Damn straight",
```
Comment (sindresorhus) [inline yes.json]: Keeping it friendly, so gonna pass on this.

**Comment by guyhughes** [comment]
ready for senpai's consideration

**Comment by sindresorhus** [comment]
Awesome. Thanks :)

**Pull request closed without merging**